official-package/terraform/README.md

```diff
@@ -0,0 +1,64 @@
+# CrowdStrike Falcon Distributor Terraform Module
+
+This terraform code will create a AWS State Manager assocation that will install the CrowdStrike Falcon Sensor on all EC2 instances in a region. The CrowdStrike Falcon Sensor is installed using the Official CrowdStrike Distributor package.
+
+By default the terraform code will deploy to all regions supported by the Official Distributor package that are enabled on your account.
```

Comment:
```suggestion
**By default the terraform code will deploy to all regions supported by the Official Distributor package that are enabled on your account.**
```

I would consider making this either bold as suggested, or use an alert like below:

> [!IMPORTANT]
> **By default the terraform code will deploy to all regions supported by the Official Distributor package that are enabled on your account.**

Reply:
Merged all those other suggestions imo this one isn't more important than the other paragraphs explaining the purpose of the module. Thoughts?

Reply:
🤷🏼‍♂️ I feel like this is one of those points where someone will undoubtedly go.. oh.. I didn't know it would deploy to all the regions by default lol.. I was just trying to highlight that this is the default.